src/main/java/com/team/buddyya/certification/controller/CertificationController.java

```diff
@@ -0,0 +1,34 @@
+package com.team.buddyya.certification.controller;
+
+import com.team.buddyya.auth.domain.CustomUserDetails;
+import com.team.buddyya.certification.dto.request.EmailCertificationRequest;
+import com.team.buddyya.certification.dto.request.EmailCodeRequest;
+import com.team.buddyya.certification.dto.response.EmailCertificationResponse;
+import com.team.buddyya.certification.service.CertificationService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/certification")
+public class CertificationController {
+
+    private final CertificationService certificationService;
+
+    @PostMapping("/email")
+    public ResponseEntity<EmailCertificationResponse> sendEmail(@AuthenticationPrincipal CustomUserDetails userDetails, @RequestBody EmailCertificationRequest emailCertificationRequest) {
+        return ResponseEntity.ok(certificationService.certificateEmail(userDetails.getStudentInfo(), emailCertificationRequest));
+    }
+
+    @PostMapping("/email/code")
+    public ResponseEntity<EmailCertificationResponse> emailCodeCertificate(@AuthenticationPrincipal CustomUserDetails userDetails, @RequestBody EmailCodeRequest codeRequest) {
+        return ResponseEntity.ok(certificationService.certificateEmailCode(userDetails.getStudentInfo(), codeRequest));
+    }
+}
```

Comment:
```suggestion
@RestController
@RequiredArgsConstructor
@RequestMapping("/certification/email")
public class CertificationController {

    private final CertificationService certificationService;

    @PostMapping("/send-email")
    public ResponseEntity<EmailCertificationResponse> sendEmail(@AuthenticationPrincipal CustomUserDetails userDetails, @RequestBody EmailCertificationRequest emailCertificationRequest) {
        return ResponseEntity.ok(certificationService.certificateEmail(userDetails.getStudentInfo(), emailCertificationRequest));
    }

    @PostMapping("/verify-code")
    public ResponseEntity<EmailCertificationResponse> emailCodeCertificate(@AuthenticationPrincipal CustomUserDetails userDetails, @RequestBody EmailCodeRequest codeRequest) {
        return ResponseEntity.ok(certificationService.certificateEmailCode(userDetails.getStudentInfo(), codeRequest));
    }
}
```

p3: 엔드포인트에 email이 겹치는 것 같아 email을 클래스 레벨의 `@RequestMapping`에 붙이는 것이 좋을 것 같아요.
추가적으로 저는 헨드폰 인증에서 엔드포인트를 설정할 때, 위와 같이 send와 verify같은 동사를 넣어서 좀 더 api의
역할을 조금 더 명확하게 하려고 했었어요. 혹시 준영님은 어떤 방식이 더 나은거 같으신가요?

Reply:
네 저도 send, verify로 네이밍 수정하도록 하겠습니다!